Documentation/QuickInstall/TheInstallTool/Index.rst

```diff
@@ -76,6 +76,21 @@ The Admin Tool
       Proceed with :ref:`install-introduction-package`
 
 
+
+After the last step of the installation you must verify the image handling settings
```

Comment:
IMHO this is the wrong location for this documentation. The next section (after the one changed here) is still on the last page of the installer. There is no way to access the config for image settings from the installer, without simultaneously completing the installer. 
I would argue that the image settings belong to https://docs.typo3.org/m/typo3/guide-installation/10.4/en-us/In-depth/TheInstallTool/Index.html#the-install-tool-in-depth (at the bottom, image handling).